### PR TITLE
source/cpu: don't create cpu-security.tdx.total_keys label

### PR DESCRIPTION
**Pull request by marquiz**
Just have that as a feature for NodeFeatureRules to consume.

### CODE REVIEW

**Comment by marquiz** [comment]
My original thinking with #1079 was to NOT have tdx.total_keys as a default built-in label. Just as a feature for NodeFeatureRule(s) to consume and expose. But I lost that idea during the review process. This PR kinda implements what I had in mind. 

@ping @fidencio @mythi @zvonkok WDYT?

**Review by fidencio — APPROVED** [review]
lgtm, thanks @marquiz!

File: source/cpu/cpu.go [hunk 1]
```diff
@@ -173,8 +173,13 @@ func (s *cpuSource) GetLabels() (source.FeatureLabels, error) {
 	}
 
 	// Security
+	skipLabel := map[string]struct{}{
```
Comment (PiotrProkop) [inline source/cpu/cpu.go:176]: you can use `k8s.io/apimachinery/pkg/util/sets` and then just:
```
sets.NewString("tdx.total_keys")
```
and check if given feature is in skipLabel by:
```
if !skipLabel.Has(k){
...
}
```

Comment (marquiz) [reply source/cpu/cpu.go]: @PiotrProkop nice suggestion, I now changed the code like that

**Comment by PiotrProkop** [comment]
lgtm, will wait for others to comment before giving proper one!

File: source/cpu/cpu.go [hunk 2]
```diff
@@ -173,8 +174,11 @@ func (s *cpuSource) GetLabels() (source.FeatureLabels, error) {
 	}
 
 	// Security
+	skipLabel := sets.NewString("tdx.total_keys")
```
Comment (mythi) [inline source/cpu/cpu.go:177]: Maybe add a comment?
```suggestion
        // A set of features that will not have labels created but are only made available for
        // NodeFeatureRules (e.g. to be published via extended resources instead)
	skipLabel := sets.NewString("tdx.total_keys")
```

Comment (marquiz) [reply source/cpu/cpu.go:179]: 👍 Added a commet

**Review by mythi — APPROVED** [review]
/lgtm

**Pull request merged**